ansible/roles/test/files/ptftests/IP_decap_test.py

```diff
@@ -263,7 +313,22 @@ def send_and_verify(self, dst_ip, expected_ports, src_port, outer_pkt='ipv4', tr
         assert received
         return matched, received
 
-    #-----------------------------------------------------------------
+    def send_and_verify_all(self, dst_ip, expected_ports, src_port, outer_pkt_type):
+        """
+        @summary: This method builds different encap packets, send and verify their arrival
+        @dest_ip: The destination ip for the inner IP header
+        @expected_ports: List of ports that a packet can arrive from
+        @src_port: The physical port that the packet will be sent from
+        @outer_pkt_type: Indicates whether the outer packet is ipv4 or ipv6
+        """
+
+        self.send_and_verify(dst_ip, expected_ports, src_port, outer_pkt_type)
+        self.send_and_verify(dst_ip, expected_ports, src_port, outer_pkt_type, outer_ttl=64, inner_ttl=2)
+        self.send_and_verify(dst_ip, expected_ports, src_port, outer_pkt_type, outer_ttl=1, inner_ttl=64)
```

Comment:
Does this (outer_ttl = 1) works if the ttl mode is set to `uniform ` since the `exp_ttl` in that case would be 0. If test fails, should we have some checks?